challenges/ArrayReverse.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Arrays;
+
+public class ArrayReverse {
+    public static void main(String[] args) {
+        System.out.println("Hello World");
+
+        System.out.println(Arrays.toString(reverseArray(new int[] { 1, 2, 4, 6, 79,6,4,8,19,89,8,2,34, })));
```

Comment:
Keep you spacing consistent. Like for this array, keep putting spaces after all the comas.

Reply:
Also, you don't need the comma after the last value of 34